02-peer-protocol.md

```diff
@@ -1381,7 +1381,7 @@ A node:
     `your_last_per_commitment_secret` is correct for that
     `next_revocation_number` minus 1:
       - MUST NOT broadcast its commitment transaction.
```

Comment:
as @rustyrussell mentioned and according to BOLT 1 upon sending an `error` the node SHOULD fail the channel  and failing usually means broadcasting the commitment transaction. 

This situation seems a little bit paradox because obviously in this case the node does not want to / MUST NOT broadcast its commitment transaction and thus can't really fail the channel. 

I wonder if this could also leave the node in a problematic situation indicating  the peer that they may attempt to cheat? But I guess it can't be worse than the already existing situation.